docs/contributing/contribution_guides/development.md

```diff
@@ -0,0 +1,89 @@
+* #### [Local Development](../LOCAL_DEVELOPMENT.md)
+
+If you don't feel ready to make a code contribution yet, no problem! You can also help out in other ways, such as [documentation](documentation.md) or [translation](translation.md).
+
+There are two paths to making a development contribution: 
+
+1. Choosing an open issue to address. Open issues can be found in the [Issues](https://github.com/invoke-ai/InvokeAI/issues?q=is%3Aissue+is%3Aopen) section of the InvokeAI repository. These are tagged by the issue type (bug, enhancement, etc.) along with the “good first issues” tag denoting if they are suitable for first time contributors.
+    1. Additional items can be found on our roadmap <******************************link to roadmap>******************************. The roadmap is organized in terms of priority, and contains features of varying size and complexity. If there is an inflight item you’d like to help with, reach out to the contributor assigned to the item to see how you can help. 
+2. Opening a new issue or feature to add. **Please make sure you have searched through existing issues before creating new ones.**
+
+*Regardless of what you choose, please post in the  [#dev-chat](https://discord.com/channels/1020123559063990373/1049495067846524939) channel of the Discord before you start development in order to confirm that the issue or feature is aligned with the current direction of the project. We value our contributors time and effort and want to ensure that no one’s time is being misspent.*
+
+## Best Practices: 
+* Keep your pull requests small. Smaller pull requests are more likely to be accepted and merged
+* Comments! Commenting your code helps reviwers easily understand your contribution
+* Use Python and Typescript’s typing systems, and consider using an editor with [LSP](https://microsoft.github.io/language-server-protocol/) support to streamline development
+* Make all communications public. This ensure knowledge is shared with the whole community
+
+## **How do I make a contribution?**
+
+Never made an open source contribution before? Wondering how contributions work in our project? Here's a quick rundown!
+
+Before starting these steps, ensure you have your local environment [configured for development](../LOCAL_DEVELOPMENT.md).
+
+1.  Find a [good first issue](https://github.com/invoke-ai/InvokeAI/contribute) that you are interested in addressing or a feature that you would like to add. Then, reach out to our team in the [#dev-chat](https://discord.com/channels/1020123559063990373/1049495067846524939) channel of the Discord to ensure you are  setup for success. 
+2. Fork the [InvokeAI](https://github.com/invoke-ai/InvokeAI) repository to your GitHub profile. This means that you will have a copy of the repository under **your-GitHub-username/InvokeAI**.
+3. Clone the repository to your local machine using:
+
+```bash
+git clone https://github.com/your-GitHub-username/InvokeAI.git
```

Comment:
I would also add that Github Desktop is a nice GUI alternative for people who are not familiar with git on the command line. Had a few people recently who started contributing but were not sure how to work with git but found the desktop app really easy to use. Expecting there to be more like that.

Reply:
Good call out - will add that!